README.adoc

```diff
@@ -122,23 +93,27 @@ Now, navigate back to the `start` directory to begin.
 
 
 // =================================================================================================
-// Enabling fault tolerance in the application
+// Enabling fault tolerance
 // =================================================================================================
 
-== Enabling fault tolerance in the application
+== Enabling fault tolerance
+
+The MicroProfile Fault Tolerance API was added as a dependency to your `pom.xml` file. Look for
+the dependency with the `org.eclipse.microprofile.fault-tolerance` group ID.
+Adding this dependency allows you to use the fault tolerance policies in your microservices.
 
-The MicroProfile Fault Tolerance API is added as a dependency to your `pom.xml` file. Look for
-the dependency with the `org.eclipse.microprofile.fault-tolerance` group ID. Use this feature
-to build fault-tolerant microservices with the MicroProfile Fault Tolerance API.
-The `mpFaultTolerance-1.0` feature is also enabled in the `src/main/liberty/config/server.xml`
-file.
+You can also find the `mpFaultTolerance-1.0` feature in your `src/main/liberty/config/server.xml` server configuration,
+which turn on MicroProfile Fault Tolerance in Open Liberty.
 
 To easily work through this guide, the two provided microservices are set up to run
 on the same server. To simulate the availability of the services and then to enable fault tolerance,
 dynamic configuration with MicroProfile Configuration is used so that you can easily take one service
 or the other down for maintenance. If you want to learn more about setting up dynamic configuration,
 see https://openliberty.io/guides/microprofile-config.html[Configuring microservices].
 
+The following two steps are setting up the dynamic configuration on the `system` service and its client.
+You can move on to the next section, which adds the fallback mechanism on the `inventory` service.
```

Comment:
How about...
"The following two steps set up the dynamic configuration on the `system` service and its client.
The next section adds the fallback mechanism on the `inventory` service."

Reply:
I want to say the couple paragraphs after this is optional, user can skip them to go to the next section, if they're not interested in how the dynamic configuration is setup. So can't get rid of the "You can move to the next section".

Reply:
OK, what about we change the first sentence while leaving the second one alone?
"The following two steps set up the dynamic configuration on the `system` service and its client.
You can move on to the next section, which adds the fallback mechanism on the `inventory` service."

Reply:
Yeap, already fixed the first sentence. Thanks.